src/euphorie/client/tests/test_model.py

```diff
@@ -17,6 +21,11 @@
     import mock
 
 
+@implementer(IClient)
+class MockContext(Acquisition.Implicit):
```

Comment:
There should be no need to have a mocked context here.
The portal should have an existing clients folder